packages/common/src/ceramic-api.ts

```diff
@@ -91,10 +99,16 @@ export interface AdminApi {
   nodeStatus(): Promise<NodeStatusResponse>
 
   /**
+   * @deprecated
    * List indexed model streams
    */
   getIndexedModels(): Promise<Array<StreamID>>
 
+  /**
+   * List indexed model streams with data
+   */
+  getIndexedModelData(): Promise<Array<ModelData>>
```

Comment:
wasn't this already added in the previous PR?  Is this PR not merged to latest develop?

Reply:
This is up to date with develop, but this wasn't in the last PR. Last PR only focused on the external API. This method is required for all internal calls.

Reply:
look on line 115, looks like the method definition wound up duplicated and included twice.  Maybe from a merge issue?  Can we remove the duplicate in a follow-up?